### DISCUSSION TITLE
Katas del día 10

### DISCUSSION THREAD

**Discussion by oscardanielM** [LaunchX-InnovaccionVirtual/CursoIntroPython#132] [General]
Hola, disculpen saben si las katas que se habilitarían el día 10 solo estarían activas para ese día? Es que yo no pude entrar el día de ayer para revisar hasta hoy y me marca esto 😢
![image](https://user-images.githubusercontent.com/36013303/153625641-3c148b97-4fe7-4c44-b1af-f03a79cab76c.png)

**Comment by ascamarena** [comment]
----> #98

**Reply by omarfrias-maker** [reply]
Se activarán el día de hoy sábado.

**Comment by AntonioFloriano** [comment]
No estuvieron activas ayer tampoco, estuve revisando y no las activaron.

**Reply by AzaelBarbosa** [reply]
No las activaron, lo mas probable que las activen el dia de hoy durante la transmision del la noche.

**Reply by Ingrzs** [reply]
Si o si en el live de hoy dirán o activarán los demás módulos, ya que tal ves no lo activan porque muchos apenas están registrándose y todos tienen que ir al mismo nivel,como dijeron en un live no puedes ir ni muy atrás ni muy adelante porque afectas al grupo

**Reply by DiegoDelgadoM** [reply]
> Hello, no era a las 8 el live?? no veo el enlace por ningún lado

ya esta en espera empieza 8:10pm

**Reply by leence** [reply]
Jojo, si ya vi

**Reply by leence** [reply]
Gracias
